interpreter/llvm/src/tools/clang/tools/driver/cc1_main.cpp

```diff
@@ -65,11 +65,13 @@ static void LLVMErrorHandler(void *UserData, const std::string &Message,
   exit(GenCrashDiag ? 70 : 1);
 }
 
-#ifdef LINK_POLLY_INTO_TOOLS
+#if 0
+#if defined(LINK_POLLY_INTO_TOOLS) && LINK_POLLY_INTO_TOOLS != 0
 namespace polly {
 void initializePollyPasses(llvm::PassRegistry &Registry);
 }
 #endif
+#endif
```

Comment:
Doesn't this affect the build with `builtin_llvm=ON`?

Reply:
No, because the our llvm config undefines polly. However the standard llvm install ships polly libraries and we end up having something like `#define LINK_POLLY_INTO_TOOLS` instead of `#define LINK_POLLY_INTO_TOOLS 0`. That a bug in clang.

Reply:
Ok, thanks for the explanation. Is this already fixed upstream? We could just add `#undef LINK_POLLY_INTO_TOOLS` at the beginning of the file, then, right?

Reply:
This is not yet fixed upstream. I will have to come up with a patch. `#undef`-ing would work (that's what our llvm-config does when polly is disabled) but it would be inconsistent with the way clang handles such options. My patch (except for the `if 0` does what other similar cases are doing).